app/src/dex/router/route_and_fill.rs

```diff
@@ -164,6 +159,7 @@ trait RouteAndFillInner: StateWrite + Sized {
 
                 tracing::debug!(?path, delta_1 = ?delta_1.amount, "found path, starting to fill up to spill price");
 
+                // TODO: in what circumstances should we use fill_route_exact?
```

Comment:
Ideally we'd have one `route_and_fill` method that we could use for both swaps and arbitrage.  I'm not sure, however, how we want to handle price limits in the two cases, relative to the changes in #2535